### DISCUSSION TITLE
LeaderBoard

### DISCUSSION THREAD

**Discussion by vijaychandra1910** [recodehive/awesome-github-profiles#615] [Q&A] [answered]
Why my rank is not showing in the leaderboard

**Comment by sanjay-kv** [comment]
share few of your merged PR here I can have a look @vijaychandra1910

**Comment by vijaychandra1910** [comment]
![image](https://github.com/user-attachments/assets/abecacd8-3991-4580-a603-0e726f12de7f)
Merged my pill request successfully but not showing in leaderboard why?

**Comment by vijaychandra1910** [comment, marked as answer]
@sanjay-kv 
![image](https://github.com/user-attachments/assets/ef71e3d2-24b9-4052-8626-9d10585feb48)

**Reply by sanjay-kv** [reply]
next time provide the link. 

as many people contribute faster response need link, now I need to find your repo from merged list which is gonna time consuming and many PA doesn't do it.
